db/blip_handler.go

```diff
@@ -932,6 +944,19 @@ func (bh *blipHandler) processRev(rq *blip.Message, stats *processRevStats) (err
 		}
 	}()
 
+	// throttle concurrent revs
+	if cap(bh.inFlightRevsThrottle) > 0 {
```

Comment:
Is the `if cap(bh.inFlightRevsThrottle) > 0 {` necessary? We always specify a non zero value and if you did have a zero value it would not hit the default case anyway?

Reply:
Thanks, makes sense.